test/e2e/e2e_test.cpp

```diff
@@ -97,7 +97,7 @@ TEST_F(E2ETest, MultiPartitionTxn) {
 }
 
 TEST_F(E2ETest, MultiHomeTxn) {
-  for (size_t i = 0; i < NUM_MACHINES; i++) {
+  for (size_t i = 0; i < 1; i++) {//NUM_MACHINES; i++) {
```

Comment:
typo?

Reply:
yes. It was when I was debugging. Nice catch!